medical-clinic/src/main/java/com/Mafiuz04/medicalclinic/service/AppointmentService.java

```diff
@@ -38,6 +50,10 @@ public AppointmentDto createAppointment(AppointmentDto appointment) {
         return appointmentMapper.toDto(entity);
     }
 
+    //TC1: W przypadku gdy wizyta o raz pacient podanych ID istnieją, pacient zostanie podpięty do wizyty, a wizyta zostanie zapisana oraz zmapowana.
```

Comment:
if (isInPast(appointment.getStartDate())) {
            throw new MedicalClinicException("Appointment is no available any more, please check date.", HttpStatus.BAD_REQUEST);
            
            nie weryfikujesz tego w tc1

Reply:
w tc2 i tc3 tez to musisz dodac